src/gui/screen_sheet_rename.cpp

```diff
@@ -29,7 +29,7 @@ screen_sheet_rename_t::screen_sheet_rename_t()
 void onclick_ok() {
     ///TODO: store count changed characters
```

Comment:
[TODOdododododo](https://www.youtube.com/shorts/Ng5qiY26Rj0)

Reply:
The functionality of sheet renaming is not yet implemented